deployment/README.md

```diff
@@ -0,0 +1,56 @@
+# Deployments
+
+## k8s Deployments
+
+### Production, Staging & Develop
+
+- lexbox
+  - .net API (GQL API, REST API, hg proxy etc. (see [backend README](../backend/README.md)))
+  - OTEL collector
+- db (PostgresDB)
+- hg
+  - hg
+  - hg-resumable
```

Comment:
everywhere except production there's a container called `populate-test-repos` that will download repos at startup